test/db/tools/rz_asm

```diff
@@ -395,6 +395,7 @@ EOF
 RUN
 
 NAME=rz-asm -A with -c and -o
+BROKEN=1
```

Comment:
Why is this so?

Reply:
Ah no. This is https://github.com/rizinorg/rizin/issues/3694. `MSR` handling in ESIL is completely off.

Reply:
Then it's fine to have it broken *but* please also add a link to the issue. Ideally, once ESIL removed this should be resolved too.